main.go

```diff
@@ -78,11 +78,14 @@ func main() {
 		}
 		progress := make(chan lib.Progress)
 		go func() {
+			// Given how the label mail counting work we are only able to render the progress
```

Comment:
This doesn't seem very desirable, though! I would imagine it's preferable to show an estimate of progress downloading mail where the error is only for messages that are members of multiple labels than to see an estimate of how many _labels_ have been synced (which will show very little incremental progress), no?